Type Error when building against Assets

Version: v7.0.0-alpha

The reference is included in Asset's index.d.ts in the but I'm not sure how it is included. It this api-extractor magic, @Zyie? Do we need to add `@types/css-font-loading-module` to the dependencies of Assets?

```
node_modules/@pixi/assets/index.d.ts:1:23 - error TS2688: Cannot find type definition file for 'css-font-loading-module'.

1 /// <reference types="css-font-loading-module" />
                        ~~~~~~~~~~~~~~~~~~~~~~~

Found 1 error in node_modules/@pixi/assets/index.d.ts:1
```

--- a/package.json
+++ b/package.json
@@ -54,7 +54,6 @@
     "@rollup/plugin-commonjs": "^15.1.0",
     "@rollup/plugin-json": "^4.1.0",
     "@rollup/plugin-node-resolve": "^9.0.0",
-    "@types/css-font-loading-module": "^0.0.7",
     "@types/jest": "^26.0.0",
     "@webdoc/cli": "^2.0.0",
     "copyfiles": "^2.1.0",

--- a/packages/assets/package.json
+++ b/packages/assets/package.json
@@ -43,6 +43,9 @@
   "bugs": {
     "url": "https://github.com/pixijs/pixi.js/issues"
   },
+  "dependencies": {
+    "@types/css-font-loading-module": "^0.0.7"
+  },
   "pixiRequirements": [
     "@pixi/core"
   ]

--- a/packages/assets/src/loader/types.ts
+++ b/packages/assets/src/loader/types.ts
@@ -4,6 +4,8 @@ export interface LoadAsset<T=any>
 {
     src: string;
     data?: T;
+    alias?: string[];
+    format?: string;
 }
 
 export interface PromiseAndParser

